CheckYourEligibility-Admin/wwwroot/js/javascript-starter.js

```diff
@@ -1,13 +1,41 @@
-document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');
-
-import { initAll } from './govuk-frontend.min.js'
-initAll();
-
-let clarityId = document.getElementsByTagName("body")[0].getAttribute("data-clarity");
-if(clarityId) {
-    (function(c,l,a,r,i,t,y){
-        c[a]=c[a]||function(){(c[a].q=c[a].q||[]).push(arguments)};
-        t=l.createElement(r);t.async=1;t.src="https://www.clarity.ms/tag/"+i;
-        y=l.getElementsByTagName(r)[0];y.parentNode.insertBefore(t,y);
-    })(window, document, "clarity", "script", clarityId);
-}
\ No newline at end of file
+const COOKIE_NAME = 'analytics-cookies-consent';
+
```

Comment:
Why has line 1 - 4 been removed? Also, document.getElementsByTagName("body") is somewhat safer than document.body I believe.